sections/variant-list.liquid

```diff
@@ -0,0 +1,182 @@
+{{ 'component-price.css' | asset_url | stylesheet_tag }}
+{{ 'variant-list.css' | asset_url | stylesheet_tag }}
+
+<script src="{{ 'variant-list.js' | asset_url }}" defer="defer"></script>
+
+{%- style -%}
+  .section-{{ section.id }}-padding {
+    padding-top: {{ section.settings.padding_top | times: 0.75 | round: 0 }}px;
+    padding-bottom: {{ section.settings.padding_bottom | times: 0.75 | round: 0 }}px;
+  }
+
+  @media screen and (min-width: 750px) {
+    .section-{{ section.id }}-padding {
+      padding-top: {{ section.settings.padding_top }}px;
+      padding-bottom: {{ section.settings.padding_bottom }}px;
+    }
+  }
+{%- endstyle -%}
+
+<variant-list
+  class="page-width section-{{ section.id }}-padding color-{{ section.settings.color_scheme }} gradient"
+  id="variant-list"
+  data-id="{{ section.id }}"
+>
+  <form class="variant__contents critical-hidden" method="post">
+    <div class="variant__items" id="main-variant-items">
+      <div class="js-contents">
+        <table class="variant-items">
+          <caption class="visually-hidden">
+            {{ 'sections.cart.title' | t }}
+          </caption>
+          <thead>
+            <tr>
+              <th class="caption-with-letter-spacing" colspan="1" scope="col">
```

Comment:
Do we need the colspans on the cell headers if we don't have any cases of colspans greater than 1? I didn't notice a difference in VO announcements, but maybe there's some impact on mobile.

Reply:
Good point!